app/services/data/get-last-claim-children.js

```diff
@@ -0,0 +1,6 @@
+const config = require('../../../knexfile').extweb
+const knex = require('knex')(config)
+
+module.exports = function (reference, eligibiltyId) {
+  return knex.raw(`SELECT * FROM [IntSchema].[getLastClaimChildren] (?, ?)`, [ reference, eligibiltyId ])
```

Comment:
only way to call SQL Server table function from knex, if we could explicitly call a function we would